document/CoreFormat.mdk

```diff
@@ -371,6 +449,46 @@ A `fileReference` object represents a single file.
 A `fileReference` object shall contain a property named `uri` whose value is a string that represents
 the location of the file as a valid URI.
 
+If this `fileReference` object refers to a file which is not nested within another file,
+then the `uri` property shall specify the location of the file, expressed as an absolute URI.
+If this object refers to a file which is nested within another file,
+then the `uri` property shall express the relative location of the file with respect to its container.
+In this case, the `uri` shall consist entirely of a fragment.
+
+If the location of the nested file with respect to its parent can be expressed
+by means of a path (as is the case for compressed containers such as Zip archives),
+then the fragment shall consist of that path, expressed as an absolute path relative to
+the root of the container.
+
+~ IsoNumberedExample { @IsoNumberedExample=1}
+
+```
+"uri": "#/src/a.cc"
+```
+~
+
+If the location of the nested file with respect to its parent cannot be expressed
+by means of a path (for example, a Flash object nested within a word processing document),
+then the fragment shall consist of a hexadecimal number specifying the offset in bytes
+of the file from the beginning of its parent container.
```

Comment:
We decided that the offset sufficed because there wouldn't be two files at the same offset from their parent.